tests/main/i18n/task.yaml

```diff
@@ -26,3 +26,14 @@ execute: |
         fi
     done
     not cat bad
+
+    echo "Lint translated commands"
+    # Only attempt to install the langpacks in Ubuntu
+    if apt install language-pack-en; then
+      # it's fine to use ls here since the filenames are very simple
+      # shellcheck disable=SC2011
+      ls /usr/share/locale/ | xargs -Iloc apt install language-pack-loc -y
+    fi
+
+    # shellcheck disable=SC2002
+    cat /usr/share/i18n/SUPPORTED | xargs -d '\n' -Iloc env SNAPD_DEBUG=1 LANG=loc snap changes everything 2>&1 > /dev/null | NOMATCH panic:
```

Comment:
nice I like this new strategy a lot better than the existing problem where users have nothing they can do about it